### PR TITLE
Remove west and south panel placement

### PR DESCRIPTION
**Pull request by jomarko**

### CODE REVIEW

**Comment by yesamer** [comment]
@jomarko Thank you, I guess this change doesn't impact on BPMN editor, right?

**Comment by jomarko** [comment]
@yesamer I am not aware of any impact for BPMN.

**Comment by yesamer** [comment]
@jomarko Thank you, not totally sure it is worth to totally remove that. Of course, I doubt we will need this again, so most likely this makes sense. BTW - I saw we have SOUTH defined as well, is that used?

**Comment by jomarko** [comment]
@yesamer I removed 'South' placement as 97495ace61

**Comment by yesamer** [comment]
@jomarko Thank you, I'll merge it just after CI build finishes with Green checks

**Comment by yesamer** [comment]
@jomarko  GWT Compilation failed. Can you please check?

**Comment by jomarko** [comment]
@yesamer thank you for spotting, pushed new commit that should solve the compilation problem.

File: packages/stunner-editors/uberfire-extensions/uberfire-simple-docks/uberfire-simple-docks-client/src/main/java/org/uberfire/client/docks/view/items/AbstractDockItem.java [hunk 1]
```diff
@@ -50,7 +50,7 @@ public abstract class AbstractDockItem extends Composite implements OuiaComponen
     public static AbstractDockItem create(UberfireDock dock,
                                           ParameterizedCommand<String> openCommand,
                                           ParameterizedCommand<String> closeCommand) {
-        if (dock.getDockPosition() == UberfireDockPosition.SOUTH) {
+        if (dock.getDockPosition() == UberfireDockPosition.EAST) {
```
Comment (yesamer) [inline packages/stunner-editors/uberfire-extensions/uberfire-simple-docks/uberfire-simple-docks-client/src/main/java/org/uberfire/client/docks/view/items/AbstractDockItem.java]: @jomarko You forgot to remove the if block here

**Review by yesamer — CHANGES_REQUESTED** [review]

**Comment by yesamer** [comment]
@jomarko What's the status of this PR? If you're focusing on other areas, we could discard this PR, as still not ready to be merged and is optional (from my perspective). WDTY?

**Comment by jomarko** [comment]
@yesamer build is green now

**Review by yesamer — APPROVED** [review]
Thank you @jomarko

**Pull request merged**